### DISCUSSION TITLE
my copilot is not working anymore

### DISCUSSION THREAD

**Discussion by Matt-Dong123** [community/community#51003] [Copilot Conversations]
### Select Topic Area

Bug

### Body

My Copilot is not working without me modifying any settings on my computer. I use its plugin in VSCode and CLion. Now on both platforms the plugin doesn't work anymore. I tried reinstalling, restarting the computer, etc. but nothing worked. After I open the output window of vscode, I can see the following log. And when I modify my code, I can see that the icon of copilot keeps rotating.What can I do to save my copilot?

![image](https://user-images.githubusercontent.com/66734553/227717908-bf3ac131-8d6e-4459-9462-b9132abf858c.png)

**Comment by ersaope** [comment]
It's possible that the issue you're experiencing with Copilot is related to its servers or the API that it uses to generate suggestions. Copilot is a machine learning tool that relies on cloud-based servers to generate suggestions, so if there's an issue with those servers or with the API, it could prevent Copilot from working properly.

You could try checking the status of the Copilot servers to see if there are any known issues that could be causing the problem. You can check the status of the Copilot servers on the GitHub status page: https://www.githubstatus.com/.

If there are no known issues with the Copilot servers, you could try resetting your API token to see if that helps. To do this, you can follow these steps:

Open the settings in VSCode or CLion
Go to the Copilot settings
Select "Reset API Token"
Generate a new API token from your GitHub account and paste it into the Copilot settings
If none of these solutions work, you can try reaching out to the Copilot support team for further assistance. They may be able to help diagnose the issue and provide a solution.
